examples/mobile/UIComponents/webclip/ads-card-2/css/style.css

```diff
@@ -1,3 +1,6 @@
 .app-banner-1 {
 	top: -17px;
+}
+.ui-footer {
```

Comment:
I think, will be better change tau less.
`.ui-card.ui-card-ads .ui-footer .ui-title`
remove margin. This margin seems not necessary.

Reply:
done